examples/cdr-select.json

```diff
@@ -14,7 +14,7 @@
       "name": "default-input",
       "tag": "select",
       "styles": ["@include cdr-select-base-mixin;"],
-      "content": "<option>default-input</option>"
+      "preContent": "<option>default-input</option>"
     }
   ]
 }
```

Comment:
It looks like you removed `select` variables above, should this also be removed until we add `select` variables?

Reply:
its commented out in the `examples/index.js` so it won't load these.

Reply:
cool, just checking.  👍